assets/js/generator/Generator.js

```diff
@@ -40,4 +40,42 @@ class Generator {
     }
 }
 
+document.addEventListener("DOMContentLoaded", function() {
+    var elems = document.querySelectorAll(".collapsible");
+    var instances = M.Collapsible.init(elems);
+});
+
+document.addEventListener("DOMContentLoaded", function() {
+    var elems = document.querySelectorAll("select");
+    var instances = M.FormSelect.init(elems);
+});
+
+/**
+ * @description: Display a box with additional data
+ */
+
+let btnBox = document.getElementById('add-attr_');
```

Comment:
Const variable in js if the variable is not modified 💯

Reply:
changed to a const